pkg/apis/core/types.go

```diff
@@ -2608,6 +2608,12 @@ type PodSecurityContext struct {
 	// +k8s:conversion-gen=false
 	// +optional
 	HostIPC bool
+	// Use host's user namespace for the pods.
+	// This field is alpha-level and can be set to false on a node only if user-namespace remapping feature is enabled at runtime and kubelet.
+	// Optional: Defaults to true, if feature-gate is enabled
+	// +k8s:conversion-gen=false
+	// +optional
+	HostUser *bool
```

Comment:
`HostUser` or `HostUserNamespace`?

Reply:
HostUser is a very generic word and can be confusing,  could we even do `HostUserNamespaceMapping` ?

Reply:
the idea of the boolean is that the uid/gids are in the host user namespace or not. adding `Mapping` makes it harder to understand what this means ("true" would actually mean there is no mapping... the uid/gid matches the host value)

Reply:
since in case of other namespaces i,e network ipc pid, "namespace" is omitted and just `hostNetwork` or `hostIPC` is used, tried to keep it consistent with existing by omitting "Namespace". Dont have a strong opinion though. If you have preference for `HostUserNamespace`, have no problem in changing to that.

Reply:
just saw other comments. Will update to hostUser